tests/installation/module_registration/register_extensions_and_modules.pm

```diff
@@ -14,18 +14,18 @@ use warnings;
 use testapi qw(save_screenshot get_var);
 
 sub run {
-    my @scc_addons = split ',', get_var('SCC_ADDONS');
+    my @scc_addons = grep($_, split(/,/, get_var('SCC_ADDONS', '')));
```

Comment:
we could use `get_required_var` so we die and not continue in case the setting is empty.

Reply:
I wondering why we needed that change, because split works find the comma at the end, but not with the comman at the beginning, but in the job setting seems to be at the end, anyway is a good addition.

Reply:
If not set 'SCC_ADDONS' setting, get_var('SCC_ADDONS', '')  this will get ''.

Reply:
there is not need for default, `get_required_var` should die if it is empty.